src/Header/Header.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Header.scss'
+
+interface HeaderName {
+    imgSource: string,
+    altTag: string,
+}
+
+function Header(props: HeaderName) {
+    return<div className='headerContainer'>
```

Comment:
I think I was imagining the navbar to be in this component, was this not done due to how Emily and Mariam have tried to blur that background of the page when the navbar is open or for another reason? Happy o leave it for now if it's not a simple case of just moving it into this component

Reply:
Yes Emily Mariam specified that navBar had to be the top component for the blurring to work correctly